app/js/main.js

```diff
@@ -964,6 +1023,20 @@ function loadMenu() {
       modifiers: "ctrl",
     }));
 
+  captureMenu.append(new nw.MenuItem({ type: "separator" }));
+
+  captureMenu.append(new nw.MenuItem({
+    label: "Play capture sounds",
+    click: function() {
+      playAudio = !playAudio;
+      notifyInfo(`Capture sounds ${playAudio ? "enabled" : "disabled"}`);
```

Comment:
I don't think we need this line. The checkbox in the Capture menu is enough indication.